static/app/components/events/interfaces/frame/frameVariables.tsx

```diff
@@ -23,15 +23,8 @@ export function FrameVariables({data, meta}: Props) {
     transformedData.push({
       key,
       subject: key,
-      value: Array.isArray(data[key])
-        ? (data[key] as unknown as any[]).map((v, i) => {
-            if (!v && meta?.[key]?.[i]?.['']) {
-              return <AnnotatedText key={key} value={v} meta={meta?.[key]?.[i]?.['']} />;
-            }
-            return v;
-          })
-        : data[key],
-      meta: meta?.[key]?.[''],
```

Comment:
I reverted this change since it looks like `KeyValueList` renders `ContextData`, which now supports showing meta on nested objects in an array / object.